assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -53,13 +68,46 @@ function FontProvider({ children }) {
 
   const maybeEnqueueFontStyle = useLoadFontFiles();
 
+  const getMenuFonts = useCallback((fontFamilyList) => {
+    const newFontList = fontFamilyList.filter(
+      (fontName) =>
+        loadedFontFamily.current.findIndex((name) => name === fontName) < 0
+    );
+    if (!newFontList?.length) {
+      return new Promise((resolve) => resolve(''));
+    }
+    return fetch(
+      `${GOOGLE_MENU_FONT_URL}?family=${encodeURI(
```

Comment:
You should use `encodeURIComponent` instead.

Reply:
Done